From df2c9b3af895a826db09ad5817f84a3aa7b6b3d9 Mon Sep 17 00:00:00 2001
From: Josh Blum <josh@joshknows.com>
Date: Tue, 2 Apr 2013 18:11:19 -0700
Subject: [PATCH 2/5] E110: revert Makefile for E110, the UCF was accidentally
 removed

---
 usrp2/top/E1x0/Makefile.E110 |    1 +
 1 file changed, 1 insertion(+)

diff --git a/usrp2/top/E1x0/Makefile.E110 b/usrp2/top/E1x0/Makefile.E110
index c2d3e39..e5be8d2 100644
--- a/usrp2/top/E1x0/Makefile.E110
+++ b/usrp2/top/E1x0/Makefile.E110
@@ -50,6 +50,7 @@ simulator "ISE Simulator (VHDL/Verilog)" \
 TOP_SRCS = \
 ../B100/u1plus_core.v \
 E1x0.v \
+E1x0.ucf \
 timing.ucf
 
 SOURCES = $(abspath $(TOP_SRCS)) $(FIFO_SRCS) \
-- 
1.7.10.4

