From f3b27b87ad49baf4696f3a62fb25dbdea9630f37 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Wed, 23 Oct 2013 14:11:58 -0700
Subject: [PATCH 2/5] Updated images downloader URL

---
 host/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 602e9aa..ff13c15 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -211,8 +211,8 @@ UHD_INSTALL(FILES
 # Images download directory for utils/uhd_images_downloader.py
 ########################################################################
 
-SET(UHD_IMAGES_MD5SUM "7eb8b0bba43cc58941115a5d3ff5c2bb")
-SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.005.004-release.zip")
+SET(UHD_IMAGES_MD5SUM "a7d0ef850779b32d3bc121dcc5a11f62")
+SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.005.004-1-gb488026e.zip")
 
 ########################################################################
 # Register top level components
-- 
1.8.4.3

