From ff1953362e77e72c73211a93b6871cc38f476a44 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Wed, 28 Feb 2018 15:44:24 -0800
Subject: [PATCH 05/40] mpm: cmake: Make default install path /usr

---
 mpm/CMakeLists.txt | 19 ++++++++++++++-----
 1 file changed, 14 insertions(+), 5 deletions(-)

diff --git a/mpm/CMakeLists.txt b/mpm/CMakeLists.txt
index 67130de4b..72d72bcfc 100644
--- a/mpm/CMakeLists.txt
+++ b/mpm/CMakeLists.txt
@@ -1,15 +1,24 @@
 #
-# Copyright 2017 Ettus Research, National Instruments Company
+# Copyright 2017-2018 Ettus Research, a National Instruments Company
 #
-# SPDX-License-Identifier: GPL-3.0
+# SPDX-License-Identifier: GPL-3.0-or-later
 #
-cmake_minimum_required(VERSION 3.1)
 
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1)
+PROJECT(MPM C CXX) # Also has Python, but CMake can take care of that later
+# Set the default value for CMAKE_INSTALL_PREFIX to /usr
+IF(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    SET(CMAKE_INSTALL_PREFIX "/usr"
+        CACHE
+        PATH
+        "Default installation path for MPM"
+        FORCE
+    )
+ENDIF(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
 ########################################################################
-# useful macros 
+# useful macros
 ########################################################################
-
 MACRO(USRP_PERIPHS_APPEND_SOURCES)
     SET(usrp_periphs_sources ${usrp_periphs_sources} PARENT_SCOPE)
     LIST(APPEND usrp_periphs_sources ${ARGV})
-- 
2.11.0

