From 12cea5a4843722daf9628006cb8b0c0580034aa7 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Mon, 3 Aug 2015 14:14:15 -0700
Subject: [PATCH 8/8] Added NI B2x0 VID/PID pairs to udev rules

---
 host/utils/uhd-usrp.rules | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/host/utils/uhd-usrp.rules b/host/utils/uhd-usrp.rules
index 2f5198d..10d68a2 100644
--- a/host/utils/uhd-usrp.rules
+++ b/host/utils/uhd-usrp.rules
@@ -1,5 +1,5 @@
 #
-# Copyright 2011 Ettus Research LLC
+# Copyright 2011,2015 Ettus Research LLC
 #
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -23,3 +23,5 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="2500", ATTRS{idProduct}=="0002", MODE:="066
 
 #B200
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="2500", ATTRS{idProduct}=="0020", MODE:="0666"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="3923", ATTRS{idProduct}=="7813", MODE:="0666"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="3923", ATTRS{idProduct}=="7814", MODE:="0666"
-- 
2.1.4

