From 1e9ac85fed2cd551183aafa3a907221b4707daa1 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Wed, 2 Sep 2015 16:02:33 -0700
Subject: [PATCH 09/13] tests: Initialize test value to avoid bit width related
 issues

---
 host/tests/sensors_c_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/tests/sensors_c_test.c b/host/tests/sensors_c_test.c
index 8babe90..9a2da33 100644
--- a/host/tests/sensors_c_test.c
+++ b/host/tests/sensors_c_test.c
@@ -116,6 +116,7 @@ int main(){
         goto free_boolean_sensor;
     }
 
+    bool_out = false;
     // Check the casted value
     UHD_TEST_EXECUTE_OR_GOTO(free_boolean_sensor,
         uhd_sensor_value_to_bool(
-- 
2.1.4

