From 92e34f4880ba033ea83409fd3cd5166ecfeba8c0 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Fri, 16 Oct 2015 14:13:45 -0700
Subject: [PATCH 10/14] max287x: assert target_freq

---
 host/lib/usrp/common/max287x.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/lib/usrp/common/max287x.hpp b/host/lib/usrp/common/max287x.hpp
index fe5da19..3d43802 100644
--- a/host/lib/usrp/common/max287x.hpp
+++ b/host/lib/usrp/common/max287x.hpp
@@ -434,6 +434,7 @@ double max287x<max287x_regs_t>::set_frequency(
     bool feedback_divided = (_regs.feedback_select == max287x_regs_t::FEEDBACK_SELECT_DIVIDED);
 
     //increase RF divider until acceptable VCO frequency (MIN freq for MAX287x VCO is 3GHz)
+    UHD_ASSERT_THROW(target_freq > 0);
     double vco_freq = target_freq;
     while (vco_freq < MIN_VCO_FREQ)
     {
-- 
2.1.4

