From 8401e84d68f2061bc651d4c89a40c31009909889 Mon Sep 17 00:00:00 2001
From: michael-west <michael.west@ettus.com>
Date: Mon, 15 Aug 2016 15:08:25 -0700
Subject: [PATCH 10/30] E100: Simplify GPSDO UART so it does not strip
 characters.

---
 host/lib/usrp/e100/e100_ctrl.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/host/lib/usrp/e100/e100_ctrl.cpp b/host/lib/usrp/e100/e100_ctrl.cpp
index 3b59a93..c164a6f 100644
--- a/host/lib/usrp/e100/e100_ctrl.cpp
+++ b/host/lib/usrp/e100/e100_ctrl.cpp
@@ -232,8 +232,6 @@ public:
 
             //got a character -> process it
             if (ret == 1){
-                if (ch == '\r') continue;
-                if (ch == '\n' and _line.empty()) continue;
                 _line += ch;
                 if (ch == '\n')
                 {
-- 
2.1.4

