From f018fa068f863d7f01887137b4bc935984430f3f Mon Sep 17 00:00:00 2001
From: Ashish Chaudhari <ashish@ettus.com>
Date: Mon, 28 Apr 2014 15:29:49 -0700
Subject: [PATCH 12/31] x300: Bumped FPGA compat number to 6.

---
 host/lib/usrp/x300/x300_fw_common.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/lib/usrp/x300/x300_fw_common.h b/host/lib/usrp/x300/x300_fw_common.h
index 1edb8b2..0bbaee3 100644
--- a/host/lib/usrp/x300/x300_fw_common.h
+++ b/host/lib/usrp/x300/x300_fw_common.h
@@ -31,7 +31,7 @@ extern "C" {
 
 #define X300_FW_COMPAT_MAJOR 3
 #define X300_FW_COMPAT_MINOR 0
-#define X300_FPGA_COMPAT_MAJOR 5
+#define X300_FPGA_COMPAT_MAJOR 6
 
 //shared memory sections - in between the stack and the program space
 #define X300_FW_SHMEM_BASE 0x6000
-- 
1.7.10.4

