From 97d338d2f8adabab13992578ece4d4b6b5d6a530 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Fri, 5 Feb 2016 13:01:49 -0800
Subject: [PATCH 13/21] Disable Winsock deprecation warnings

---
 host/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 8a3d661..3a5cae4 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -162,6 +162,7 @@ IF(MSVC)
         -D_CRT_SECURE_NO_DEPRECATE
         -D_CRT_NONSTDC_NO_WARNINGS
         -D_CRT_NONSTDC_NO_DEPRECATE
+        -D_WINSOCK_DEPRECATED_NO_WARNINGS
     )
     ADD_DEFINITIONS(/MP) #multi-threaded build
     ADD_DEFINITIONS(/bigobj) #Increases the number of addressable sections in an .obj file.
-- 
2.1.4

