From 60b656fd2dbfe3bd3f1ee939f79792f55b898ba3 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Wed, 2 Apr 2014 08:14:48 -0700
Subject: [PATCH 14/15] cmake: updated uhd_images_downloader URL

---
 host/CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 45d0cd9..c450585 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -210,8 +210,8 @@ UHD_INSTALL(FILES
 # Images download directory for utils/uhd_images_downloader.py
 ########################################################################
 
-SET(UHD_IMAGES_MD5SUM "00784ebb5243b0abb15db305f557e230")
-SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.000-48-ge1c32905.zip")
+SET(UHD_IMAGES_MD5SUM "02291061d617c7b398a0dbdcc11ca6b6")
+SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.000-70-gfcc85c95.zip")
 
 ########################################################################
 # Register top level components
-- 
1.7.10.4

