From 9a05ef5225d9d52f693ffde4ae77e6ae0b16fc6e Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz.fischer@ettus.com>
Date: Tue, 9 Feb 2016 11:16:52 -0800
Subject: [PATCH 14/21] docs: Added paragraph on dhcp timeout for e3xx

Signed-off-by: Moritz Fischer <moritz.fischer@ettus.com>
---
 host/docs/usrp_e3x0.dox | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/host/docs/usrp_e3x0.dox b/host/docs/usrp_e3x0.dox
index bdcfe8b..174a83d 100644
--- a/host/docs/usrp_e3x0.dox
+++ b/host/docs/usrp_e3x0.dox
@@ -655,6 +655,20 @@ iface eth0 inet dhcp
     hostname your-hostname
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Note: In rare occasions it might be necessary to increase the timeout value
+for the dhcp client running on the device in order for autoconfiguration
+to succeed.
+
+In order to increase the timeout to e.g. 40 seconds edit:
+
+    /etc/network/interfaces
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+auto eth0
+iface eth0 inet dhcp
+    hostname your-hostname
+    udhcpc_opts -t 40
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 \subsection e3xx_network_static Static IP
 
-- 
2.1.4

