From 62afbfb7ed4ef441dd4df66db4a1741b927a77de Mon Sep 17 00:00:00 2001
From: Moritz Fischer <moritz.fischer@ettus.com>
Date: Tue, 9 Feb 2016 12:16:33 -0800
Subject: [PATCH 15/21] docs: Added link to e3xx sdks on files.ettus.com

Signed-off-by: Moritz Fischer <moritz.fischer@ettus.com>
---
 host/docs/usrp_e3x0.dox | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/docs/usrp_e3x0.dox b/host/docs/usrp_e3x0.dox
index 174a83d..64e7843 100644
--- a/host/docs/usrp_e3x0.dox
+++ b/host/docs/usrp_e3x0.dox
@@ -86,7 +86,7 @@ You should be presented with a shell similar to the following
 
 \section e3x0_sdk Using the SDK
 
-In order to facilitate software development for the integrated ARM Cortex-A9 processor, a <a href="http://www.yoctoproject.org">Yocto Project</a> based SDK is provided in the download section of our website.
+In order to facilitate software development for the integrated ARM Cortex-A9 processor, a <a href="http://www.yoctoproject.org">Yocto Project</a> based SDK is provided in the download section of our <a href="http://files.ettus.com/e3xx_images">website</a>.
 This SDK contains a cross-compiler, a cross-linker as well as a cross-debugger and can be used to develop your user space applications for the Ettus USRP-E310 devices.
 
 
-- 
2.1.4

