From 176024de72a8fed7846c9e4405c77af7e0ff5270 Mon Sep 17 00:00:00 2001
From: Derek Kozel <derek.kozel@ettus.com>
Date: Mon, 8 Feb 2016 12:16:01 -0800
Subject: [PATCH 16/21] Update docs with lack of UBX phase sync on N210

---
 host/docs/sync.dox | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/host/docs/sync.dox b/host/docs/sync.dox
index 1789a70..5a0870b 100644
--- a/host/docs/sync.dox
+++ b/host/docs/sync.dox
@@ -158,11 +158,11 @@ Using timed commands, multiple frontends can be tuned at a specific
 time. This timed-tuning ensures that the phase offsets between VCO/PLL
 chains will remain constant after each re-tune. See notes below:
 
+-   Phase synchronization with the UBX is only supported on the X3x0 Series
+-   Phase synchronization with the SBX works on both N2x0 and X3x0 Series
 -   There is a random phase offset between any two frontends
 -   This phase offset is different for different LO frequencies
 -   This phase offset remains constant after retuning
-    -   Due to a divider, UBX phase offset will be randomly +/- 180 deg after re-tune on N200/N210.
-        On X300/X310, phase sync with UBX fully works.
 -   This phase offset will drift over time due to thermal and other characteristics
 -   Periodic calibration will be necessary for phase-coherent applications
 
-- 
2.1.4

