From 36c8e0f81addfc38441dad6bbd56e8f60574ba4d Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Tue, 17 Nov 2015 09:17:02 -0800
Subject: [PATCH 17/17] examples: Fixed error code variable in rx_samples_c

---
 host/examples/rx_samples_c.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/examples/rx_samples_c.c b/host/examples/rx_samples_c.c
index abea812..0ecb891 100644
--- a/host/examples/rx_samples_c.c
+++ b/host/examples/rx_samples_c.c
@@ -222,7 +222,7 @@ int main(int argc, char* argv[])
         EXECUTE_OR_GOTO(close_file,
             uhd_rx_metadata_error_code(md, &error_code)
         )
-        if(return_code != UHD_RX_METADATA_ERROR_CODE_NONE){
+        if(error_code != UHD_RX_METADATA_ERROR_CODE_NONE){
             fprintf(stderr, "Error code 0x%x was returned during streaming. Aborting.\n", return_code);
             goto close_file;
         }
-- 
2.1.4

