From 66eb729a768be852014154571112465dd15672bc Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Fri, 9 Mar 2018 15:36:55 -0800
Subject: [PATCH 20/40] Update CHANGELOG

---
 CHANGELOG | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/CHANGELOG b/CHANGELOG
index b3bbc63c9..87ff944ae 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -1,6 +1,15 @@
 Change Log for Releases
 ==============================
 
+
+* N300: Added driver support (includes some refactoring of N3xx
+  codebase)
+* MPM: Fix PyLint warnings, Fix error handling for TCA communication
+  errors, Fix printout of AD9371 version
+* uhd_images_downloader: Create unique archive names for images archives
+  (now include git hashes in the filename)
+* UHD: Minor logging fixes
+
 ## 003.011.000.000
 * N310: Added driver
 * UBX: Add support for CAL antenna for Rev E
-- 
2.11.0

