From 5de0bfce3f03cc45a1eed93dc1b8df1b188b5040 Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Wed, 14 May 2014 10:34:52 -0700
Subject: [PATCH 21/31] cmake: updated uhd_images_downloader URL/md5sum

---
 host/CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index b53e620..1973f7b 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -210,8 +210,8 @@ UHD_INSTALL(FILES
 # Images download directory for utils/uhd_images_downloader.py
 ########################################################################
 
-SET(UHD_IMAGES_MD5SUM "6a51f54fb440860f3023fc888c6a5898")
-SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.001-release.zip")
+SET(UHD_IMAGES_MD5SUM "fb5b36972c453e5b7954879873bbcf74")
+SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.001-20-g468e7f35.zip")
 
 ########################################################################
 # Register top level components
-- 
1.7.10.4

