From 1e803ff6ac2ac86acaeb4d4b560846b0aa85fce2 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Thu, 3 Mar 2016 11:54:06 -0800
Subject: [PATCH 23/27] x300: Added power cycle message to uhd_image_loader

---
 host/lib/usrp/x300/x300_image_loader.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/host/lib/usrp/x300/x300_image_loader.cpp b/host/lib/usrp/x300/x300_image_loader.cpp
index 9ec8a2e..e8a1b52 100644
--- a/host/lib/usrp/x300/x300_image_loader.cpp
+++ b/host/lib/usrp/x300/x300_image_loader.cpp
@@ -365,6 +365,7 @@ static void x300_ethernet_load(x300_session_t &session){
         }
         else std::cout << "successful." << std::endl;
     }
+    std::cout << str(boost::format("Power-cycle the USRP %s to use the new image.") % session.dev_addr.get("product", "")) << std::endl;
 }
 
 static void x300_pcie_load(x300_session_t &session){
@@ -382,6 +383,7 @@ static void x300_pcie_load(x300_session_t &session){
         niusrprio::nirio_status_to_exception(status, "NI-RIO reported the following error:");
     }
     else std::cout << "successful." << std::endl;
+    std::cout << str(boost::format("Power-cycle the USRP %s to use the new image.") % session.dev_addr.get("product", "")) << std::endl;
 }
 
 static bool x300_image_loader(const image_loader::image_loader_args_t &image_loader_args){
-- 
2.1.4

