From fa9e37dc0ad2f0b40d242289965438d498984bed Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Mon, 10 Mar 2014 11:55:00 +0100
Subject: [PATCH 23/39] x300: Added comments on mb/chan search code

---
 host/lib/usrp/x300/x300_io_impl.cpp |    4 ++++
 1 file changed, 4 insertions(+)

--- a/host/lib/usrp/x300/x300_io_impl.cpp
+++ b/host/lib/usrp/x300/x300_io_impl.cpp
@@ -346,6 +346,7 @@
     boost::shared_ptr<sph::recv_packet_streamer> my_streamer;
     for (size_t stream_i = 0; stream_i < args.channels.size(); stream_i++)
     {
+        // Find the mainboard and subdev that corresponds to channel args.channels[stream_i]
         const size_t chan = args.channels[stream_i];
         size_t mb_chan = chan, mb_index;
         for (mb_index = 0; mb_index < _mb.size(); mb_index++) {
@@ -358,6 +359,7 @@
             }
         }
 
+        // Find the DSP that corresponds to this mainboard and subdev
         mboard_members_t &mb = _mb[mb_index];
         radio_perifs_t &perif = mb.radio_perifs[mb_chan];
 
@@ -527,6 +529,7 @@
     boost::shared_ptr<sph::send_packet_streamer> my_streamer;
     for (size_t stream_i = 0; stream_i < args.channels.size(); stream_i++)
     {
+        // Find the mainboard and subdev that corresponds to channel args.channels[stream_i]
         const size_t chan = args.channels[stream_i];
         size_t mb_chan = chan, mb_index;
         for (mb_index = 0; mb_index < _mb.size(); mb_index++) {
@@ -538,6 +541,7 @@
                 mb_chan -= curr_subdev_spec.size();
             }
         }
+        // Find the DSP that corresponds to this mainboard and subdev
         mboard_members_t &mb = _mb[mb_index];
         radio_perifs_t &perif = mb.radio_perifs[mb_chan];
 
