From 98f8ec4e14ab92cd4891fb9986479b6c6ff88782 Mon Sep 17 00:00:00 2001
From: Ben Hilburn <ben.hilburn@ettus.com>
Date: Thu, 22 May 2014 18:04:43 -0700
Subject: [PATCH 28/31] UHD: Addressing Bug 466, as per Balint.

---
 host/lib/convert/convert_impl.cpp |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/host/lib/convert/convert_impl.cpp b/host/lib/convert/convert_impl.cpp
index c7907ed..caab392 100644
--- a/host/lib/convert/convert_impl.cpp
+++ b/host/lib/convert/convert_impl.cpp
@@ -144,4 +144,7 @@ UHD_STATIC_BLOCK(convert_register_item_sizes){
     convert::register_bytes_per_item("s32", sizeof(boost::int32_t));
     convert::register_bytes_per_item("s16", sizeof(boost::int16_t));
     convert::register_bytes_per_item("s8", sizeof(boost::int8_t));
+
+    //register VITA types
+    convert::register_bytes_per_item("item32", sizeof(boost::int32_t));
 }
-- 
1.7.10.4

