From 39127b6e4cd5e8d1fc4f27c0ef7d002545f4cf8d Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Wed, 27 Jul 2016 10:30:56 -0700
Subject: [PATCH 32/42] usrp_e3x0.dox: Example cmake line should enable GPSD
 for E310. (#1170)

Signed-off-by: Philip Balister <philip@opensdr.com>
---
 host/docs/usrp_e3x0.dox | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/docs/usrp_e3x0.dox b/host/docs/usrp_e3x0.dox
index 1a94e6c..fdaea80 100644
--- a/host/docs/usrp_e3x0.dox
+++ b/host/docs/usrp_e3x0.dox
@@ -135,7 +135,7 @@ have custom UHD modifications.
 -# Setup your environment as described in \ref e3x0_sdk_usage
 -# Type the following in the build directory (assuming a build in host/build):
 
-        $ cmake -DCMAKE_TOOLCHAIN_FILE=../host/cmake/Toolchains/oe-sdk_cross.cmake -DCMAKE_INSTALL_PREFIX=/usr -DENABLE_E300=ON ..
+        $ cmake -DCMAKE_TOOLCHAIN_FILE=../host/cmake/Toolchains/oe-sdk_cross.cmake -DCMAKE_INSTALL_PREFIX=/usr -DENABLE_E300=ON -DENABLE_GPSD=ON ..
         $ make
 
 For instructions on building UHD on a PC to interact with your E-Series device, follow these instructions: \ref e3x0_uhd_build
-- 
2.1.4

