From b01bdc552902318c35095e00708420752ac04031 Mon Sep 17 00:00:00 2001
From: Martin Braun <martin.braun@ettus.com>
Date: Fri, 12 Aug 2016 10:14:18 -0700
Subject: [PATCH 37/42] Updated submodule pointer to track UHD-3.9.LTS branch

---
 .gitmodules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitmodules b/.gitmodules
index 473a212..b625125 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,4 +1,4 @@
 [submodule "fpga-src"]
 	path = fpga-src
 	url = https://github.com/EttusResearch/fpga.git
-	branch = maint
+	branch = UHD-3.9.LTS
-- 
2.1.4

