From d49775cf10699a902b283b7721e0100ec7d7c69a Mon Sep 17 00:00:00 2001
From: Nicholas Corgan <nick.corgan@ettus.com>
Date: Mon, 24 Mar 2014 14:49:59 -0700
Subject: [PATCH 38/39] cmake: updated images downloader URL and MD5sum

---
 host/CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 28ad6b9..45d0cd9 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -210,8 +210,8 @@ UHD_INSTALL(FILES
 # Images download directory for utils/uhd_images_downloader.py
 ########################################################################
 
-SET(UHD_IMAGES_MD5SUM "596a2476126bb5fd792ec685d88f8365")
-SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.000-release.zip")
+SET(UHD_IMAGES_MD5SUM "00784ebb5243b0abb15db305f557e230")
+SET(UHD_IMAGES_DOWNLOAD_SRC "http://files.ettus.com/binaries/maint_images/archive/uhd-images_003.007.000-48-ge1c32905.zip")
 
 ########################################################################
 # Register top level components
-- 
1.7.10.4

