From 3939fa90a99749b452e644ed0848eaa5133deef2 Mon Sep 17 00:00:00 2001
From: michael-west <michael.west@ettus.com>
Date: Fri, 16 Mar 2018 16:13:05 -0700
Subject: [PATCH 39/40] cmake: Revert unintentional change in UHDVersion.cmake

---
 host/cmake/Modules/UHDVersion.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/host/cmake/Modules/UHDVersion.cmake b/host/cmake/Modules/UHDVersion.cmake
index 658b9b1c4..0b8739823 100644
--- a/host/cmake/Modules/UHDVersion.cmake
+++ b/host/cmake/Modules/UHDVersion.cmake
@@ -21,8 +21,7 @@ SET(UHD_VERSION_MAJOR   3)
 SET(UHD_VERSION_API    11)
 SET(UHD_VERSION_ABI     0)
 SET(UHD_VERSION_PATCH   0)
-SET(UHD_VERSION_DEVEL TRUE)
+SET(UHD_VERSION_DEVEL FALSE)
 
 ########################################################################
 # If we're on a development branch, we skip the patch version
-- 
2.11.0

